bokeh/core/compat/bokeh_renderer.py

```diff
@@ -147,7 +147,10 @@ def open_axes(self, ax, props):
 
     def close_axes(self, ax):
         "Complete the axes adding axes-dependent plot props"
-        background_fill_color = convert_color(ax.get_axis_bgcolor())
+        if LooseVersion(mpl.__version__) <= '1.5.9':
```

Comment:
I'm never sure what LooseVersion means, so I compared with a never happening release. 

Yes, this is to avoid deprecation warnings before they reach the end-user eye.